src/grt/grt-wave_opt_file-parse-debug.adb

```diff
@@ -39,22 +42,24 @@ package body Grt.Wave_Opt_File.Parse.Debug is
          else
             Put_Line ("Instances : ");
          end if;
-         Dump_Sub_Tree (Trees (Index), 1);
+         Dump_Sub_Tree (Trees (Index));
       end loop;
       Put_Line ("----------- END -----------------");
       New_Line;
    end Dump_Tree;
 
-   procedure Dump_Sub_Tree (Cursor : Elem_Acc; Level : Positive)
+-------------------------------------------------------------------------------
```

Comment:
Ok, then I will fix it some time in the future.


Le 28. 07. 16 à 07:46, tgingold a écrit :
>
> In src/grt/grt-wave_opt_file-parse-debug.adb 
> <https://github.com/tgingold/ghdl/pull/124#discussion_r72567179>:
>
> >        end loop;
> >        Put_Line ("----------- END -----------------");
> >        New_Line;
> >     end Dump_Tree;
> >
> > -   procedure Dump_Sub_Tree (Cursor : Elem_Acc; Level : Positive)
> > +-------------------------------------------------------------------------------
>
> I prefer not to impose an order between 'public' and 'private' 
> subprograms as it is often clearer to interleave them (because you 
> group related subprograms). That's why I usually don't separate them.
>
> I am maybe too picky on style convention and I know this is not the 
> most important point.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/tgingold/ghdl/pull/124/files/eae288207d481e422e395aa7d310fa629d20432b#r72567179>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/ATNePnHgNN4saUSuMh21C01UZUfHH83Xks5qaEJJgaJpZM4JVgTD>.
>